components/file_attachment/__snapshots__/file_attachment.test.jsx.snap

```diff
@@ -9,8 +9,17 @@ exports[`component/FileAttachment should match snapshot, after change from file
     href="#"
     onClick={[Function]}
   >
-    <div
-      className="post-image__load"
```

Comment:
This should be the snapshot. This was caused by the enzyme issue with 3.9.0 airbnb/enzyme#2027